src/subspace_store.c

```diff
@@ -116,6 +116,7 @@ ts_subspace_store_add(SubspaceStore *store, const Hypercube *hc, void *object,
 			 * create one now. (There will always be one for time)
 			 */
 			Assert(last != NULL);
+			/* NOLINTNEXTLINE(clang-analyzer-core.NullDereference) */
```

Comment:
Does it makes sense to create compatible macroses for those tweaks/comments? Hopefully they will still work :)